source/access/text_mode_access_using_putty.rst

```diff
@@ -28,42 +38,85 @@ VSC clusters.
       <hardware>`, and replace ``login.hpc.kuleuven.be`` accordingly.
 
 
-#. Within the category Session, in the field 'Host Name', type in
-   <vsc-loginnode>, which is the name of the login node of the VSC
-   cluster you want to connect to.
+- Within the category 'Session', in the field 'Host Name', type in
+  a valid hostname of the :ref:`login node of the VSC cluster <hardware>`
+  you want to connect to.
 
    .. figure:: text_mode_access_using_putty/text_mode_access_using_putty_01.png
 
-#. In the category Connection > Data, in the field 'Auto-login
-   username', put in <vsc-account>, which is your VSC username that you
-   have received by mail after your request was approved.
+- In the category Connection > Data, in the field 'Auto-login
+  username', put in <vsc-account>, which is your VSC username that you
+  have received by mail after your request was approved.
 
    .. figure:: text_mode_access_using_putty/text_mode_access_using_putty_02.png
 
-#. In the category Connection > SSH > Auth > Credentials, click on 'Browse' and select
-   the private key that you generated and saved above.
+- Based on the destination VSC site that you want to login to, choose one of the
+  tabs below and proceed.
 
-   .. figure:: text_mode_access_using_putty/text_mode_access_using_putty_04.png
 
-   Here, the private key was previously saved in the folder
-   ``C:\Users\Me\Keys``. In older versions of Windows, you would have
-   to use ``C:\Documents and Settings\Me\Keys``.
+  .. tab-set::
 
-#. In the category Connection > SSH > X11, click the Enable X11
-   Forwarding checkbox:
+     .. tab-item:: KU Leuven
 
-   .. figure:: text_mode_access_using_putty/text_mode_access_using_putty_05.png
+        Select the SSH > Auth > Credentials' tab, and remove any private key from the
+        box 'Private key file for authentication'.
 
-#. In the category Connection > SSH > Auth, you can enable agent forwarding
-   by ticking the 'Allow agent forwarding' checkbox.
+        .. _putty_auth_panel:
+        .. figure:: text_mode_access_using_putty/putty_priv_key.PNG
+           :alt: putty private key
+
+     .. tab-item:: UGent, VUB, UAntwerpen
+
+        In the category Connection > SSH > Auth > Credentials, click on 'Browse',
+        and select the private key that you generated and saved above.
+
+        .. figure:: text_mode_access_using_putty/text_mode_access_using_putty_04.png
+
+        Here, the private key was previously saved in the folder
+        ``C:\Users\Me\Keys``.
+        In older versions of Windows, you would have to use
+        ``C:\Documents and Settings\Me\Keys``.
+
+- In the category Connection > SSH > Auth, make sure that the option
+  'Attept authentication using Pageant' is selected.
+  It is also recommended to enable agent forwarding by ticking the
+  'Allow agent forwarding' checkbox.
 
    .. figure:: text_mode_access_using_putty/text_mode_access_using_putty_03.png
```

Comment:
suggestion in connection to https://github.com/hpcleuven/VscDocumentation/pull/438/files#r1867410667 (part II):
```suggestion
```